ci/Jenkinsfiles/release.groovy

```diff
@@ -273,6 +284,10 @@ pipeline {
           currentBuild.description = "Release ${RELEASE_VERSION} from build ${params.BUILD_VERSION}"
         }
       }
+      slackSend(channel: "${SLACK_CHANNEL}", color: "good", message: "Successfully promoted ${BRANCH_NAME} from ${CURRENT_VERSION} to ${RELEASE_VERSION}: ${BUILD_URL}")
```

Comment:
Should be it in the !DRY_RRUN block, no?

Reply:
Same thing I would go for "Successfully released ${RELEASE_VERSION} from build ${params.BUILD_VERSION}: ${BUILD_URL}"

Reply:
In fact I didn't want to remove it from the suggestion for the build.
Let's always display the build link: success/unsuccessful/build/release.